### ISSUE TITLE
Version doesn't download

### ISSUE THREAD

**Issue by azukaar** [NobodyMiner/minecraft-client#1]
Fantastic work you got here.
I tried a simple code, but I seems to be missing something as it's not downloading the version. And even if I provide it myself, it claims it can't "find or load main class".
This is on windows BTW.
Thanks for your help

```
let {MinecraftClient, Authentication, InstallationProgress} = require('minecraft-client');

//Vanilla Minecraft Client
MinecraftClient.getMinecraftClient("1.13.1", {
    gameDir: 'C:\\Users\\azuka\\AppData\\Roaming\\.minecraft'
}, InstallationProgress.callback(() => {
  console.log('step')
}, (progress) => {
  console.log(progress)
})).then((client) => {
  client.launch(Authentication.offline("someone"));
})
```

**Comment by azukaar** [comment]
BTW, looking at the code I found the `checkInstallation` and calling it first makes the download work, but then the downloaded instance still shows ` Could not find or load main class`

**Comment by azukaar** [comment]
This way it works :

```
let {MinecraftClient, Authentication, InstallationProgress} = require('minecraft-client');


//Vanilla Minecraft Client
MinecraftClient.getForgeClient("1.12.2", "recommended", {
    gameDir: 'C:\\Users\\azuka\\AppData\\Roaming\\.minecraft'
}, InstallationProgress.callback(() => {
  console.log('step')
}, (progress) => {
  console.log(progress)
})).then((client) => {
  client.checkInstallation().then(() => {
    client.launch(Authentication.offline("someone"));
  })
});
```

**Comment by xcf-t** [comment]
The `client.checkInstallation()` is required to run before launch. Is the problem with `Could not find or load main class` still occurring after running it? If it does please send the full console output.

**Comment by xcf-t** [comment]
I just found out that I missed the `checkInstallation` in the documentation. Sorry, I'm new to doing public projects...

Edit:
Hopefully fixed in 3dcb6849d3927ed8167167f9e8f710c295bbf424

**Comment by azukaar** [comment]
Thanks you ! Yes it does fix the issue
